src/app/components/Tokens/TokenLinkWithIcon.tsx

```diff
@@ -18,9 +18,6 @@ export const TokenLinkWithIcon: FC<{
   return (
     <div className="flex items-center gap-2">
       <Tooltip
-        placement="top"
-        arrow
-        slotProps={{ tooltip: { sx: { fontWeight: 'normal' } } }}
         title={
           metadata?.origin && (
```

Comment:
In MUI this didn't show a tooltip if origin was missing:

https://explorer.dev.oasis.io/mainnet/sapphire/address/0x3cAbbe76Ea8B4e7a2c0a69812CBe671800379eC8
<img width="640" height="376" alt="Image" src="https://github.com/user-attachments/assets/6e9b7ef1-f387-4385-827b-d9662304016e" />